mpi-proxy-split/mpi-wrappers/mpi_wrappers.cpp

```diff
@@ -108,11 +108,19 @@ USER_DEFINED_WRAPPER(int, Init_thread, (int *) argc, (char ***) argv,
 // FIXME: See the comment in the wrapper function, defined
 // later in the file.
 // DEFINE_FNC(int, Finalize, (void))
-DEFINE_FNC(double, Wtime, (void))
 DEFINE_FNC(int, Finalized, (int *) flag)
 DEFINE_FNC(int, Get_processor_name, (char *) name, (int *) resultlen)
 DEFINE_FNC(int, Initialized, (int *) flag)
 
```

Comment:
I think you need to add:
`#include <time.h>`
at the top of the file (above `#include mpi.h`).

Reply:
Yes, have it added.